docs/dask_cudf/source/conf.py

```diff
@@ -83,6 +83,8 @@
 numpydoc_class_members_toctree = False
 numpydoc_attributes_as_param_list = False
 
+nitpicky = True
```

Comment:
docs for this: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-nitpicky